src/pages/resultsView/ResultsViewURLWrapper.ts

```diff
@@ -389,18 +401,37 @@ export default class ResultsViewURLWrapper
         );
     }
 
-    @computed public get comparisonSubTabId() {
-        return this.query.comparison_subtab || GroupComparisonTab.OVERLAP;
-    }
-
     @autobind
     public setTabId(tabId: ResultsViewTab, replace?: boolean) {
         this.updateURL({}, `results/${tabId}`, false, replace);
     }
 
+    public get getSubTab() {
```

Comment:
the function name should not repeat `get`

```suggestion
    public get subTab() {
```

Reply:
Done.